packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/RelationModal.tsx

```diff
@@ -198,8 +249,18 @@ const RelationModalWrapper = ({ relation, triggerButtonLabel }: RelationModalPro
                         withTooltip={false}
                         label="Back"
                         variant="ghost"
-                        disabled
-                        onClick={() => {}}
+                        disabled={!hasHistory}
+                        onClick={() => {
+                          setActionPosition('back');
+                          if (modified && !isSubmitting) {
+                            setIsConfirmationOpen(true);
+                          } else {
+                            const previousRelation = getPreviousDocument();
+                            if (previousRelation) {
+                              changeDocument(previousRelation);
+                            }
```

Comment:
this code is used in different places so it could be a function (https://github.com/strapi/strapi/pull/23074#discussion_r1982984288)